lgc/include/lgc/util/BuilderBase.h

```diff
@@ -77,6 +77,12 @@ class BuilderBase : public BuilderCommon {
   // @param passthroughArgs : The arguments to pass-through without massaging.
   llvm::Value *CreateMapToInt32(MapToInt32Func mapFunc, llvm::ArrayRef<llvm::Value *> mappedArgs,
                                 llvm::ArrayRef<llvm::Value *> passthroughArgs);
+
+  // Create an inline assembly call to cause a side effect (used to work around miscompiles with convergent).
+  llvm::Value *createInlineAsmSideEffect(llvm::Value *const value);
```

Comment:
Now that this is a proper Builder* method, can you make its name CreateInlineAsmSideEffect with a capital C to match other Builder* methods please? Same with CreateSetInactive. You might get a clang-tidy failure from that; one way to fix that would be to add these two new method names to the MethodIgnoredRegexp list in .clang-tidy.

Reply:
Thanks, I did not notice that. Have fixed in new version.